doc/api/deprecations.md

```diff
@@ -95,12 +95,18 @@ methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
 <a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
+### DEP0007: Replace cluster worker.suicide with worker.exitedAfterDisconnect
```

Comment:
Unfortunately I think it's necessary for context, otherwise those seeing the deprecation error and not having any context won't really have a clue what it is about, or any ability to find out. The whole reason `deprecations.md` exists is to provide that context when necessary.